src/angular-cesium-entities-editor/components/circles-editor/circles-editor.component.html

```diff
@@ -0,0 +1,23 @@
+<ac-layer #editPointsLayer acFor="let point of editPoints$" [context]="this" [zIndex]="1">
+    <ac-point-desc props="{
+        position: point.getPosition(),
+        pixelSize: getPointSize(point),
+        color: Cesium.Color.WHITE,
```

Comment:
get color and other props from config options,
check out what I did with polygons after i will enter my PR

Reply:
if you think you dont have enough time for it we can consider doing it in another task...

Reply:
id rather move to dynamic layer. another task